omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/dex/DexSpec.groovy

```diff
@@ -60,18 +62,18 @@ class DexSpec extends BaseRegTestSpec {
         offerTx.propertyid == currencyOffered.getValue()
 
         where: "the currency identifier is either MSC or TMSC"
-        currencyOffered << [CurrencyID.MSC, CurrencyID.TMSC]
+        currencyOffered << [MSC, TMSC]
```

Comment:
Good point.

When we do the token rebrand to Omni and Test Omni, we'll also need to rename the ecosystems. Hopefully at that point the names will allow for static import without conflict.